proprietary/Components/src/denhaag/cta-image-content.tokens.json

```
@@ -17,14 +17,22 @@
         "color": { "value": "{denhaag.color.grey.5}" },
         "font-size": { "value": "2rem" },
         "font-weight": { "value": "700" },
-        "line-height": { "value": "2.563rem" }
+        "line-height": { "value": "2.563rem" },
+        "small": {
+          "font-size": { "value": "1.5rem" },
+          "line-height": { "value": "1.938rem" }
```

I think these line-height values are incorrect. According to the design, the line-height for a small variant is 31.2 pixels. Given a font-size of 24 pixels, that would give a line-height of `1.3`.
```suggestion
          "line-height": { "value": "1.3" }
```